messages/descriptions/sensor.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Sensor description",
+    "description": "A description of a sensor including metadata of the sensor itself and produced observations",
+    "required": [
+        "sensor_id",
+        "observations"
+    ],
+    "properties": {
+        "sensor_id": {
+            "type": "string",
+            "title": "Physical sensor id",
+            "desciption": "Id of the reporting physical sensor equipment"
+        },
+        "metadata": {
+            "type": "object",
+            "title": "Metadata",
+            "description": "Metadata of this platform"
```

Comment:
So here is inserted the sensor's settings and state? Such examples range, powered on or off, alarms...  

Maybe alarms will have a totally separate structure under observations?

Reply:
According to me, if it was a GPS, the metadata would be:
"manufacturer": "ANAVS"
"model":"SuperAccurate V2.0"

The state of the sensor (range, alarms, etc.) should be part of the observations I think.